src/hw17/ProductType.java

```diff
@@ -0,0 +1,19 @@
+package hw17;
+
+public enum ProductType {
+    FRUIT("Fruit"),
```

Comment:
You may use simple, without additional fields, Enum like FRUIT, BOOK, .... up to you